### PR TITLE
Adds global styles for layout and icons

### LINKED ISSUE #54: implement neat and basic layout styles
Based on recommendations in issue #4 , initially implement [Neat](https://github.com/thoughtbot/neat) grid framework while we compile styles because we use this in rockarch.org and DIMES. 

Also add styles for container, wrapper, and gutters.

### PR DESCRIPTION
**Pull request by HaSistrunk**
Fixes #54 and #53 

- Adds Neat (this is preliminary, and will need to be reviewed as part of issue #4 after we add component styles).
- Adds global styles for CSS helper classes including `.container`, `.container--full-width`, `.wrapper`, and `.visually-hidden`. 
- Adds gutters variable and replaces boilerplate breakpoints with breakpoints used in dimes.
- Imports Material Icons font and adds an icon component with associated styles.

### CODE REVIEW

File: .stylelintrc.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "stylelint-config-sass-guidelines",
     "stylelint-config-css-modules"
   ],
-  "ignoreFiles": ["*/vendors/*.*"],
+  "ignoreFiles": ["*/vendors/**/**/**.*"],
```
Comment (helrond) [inline .stylelintrc.json]: Is there a reason we're not just ignoring the entire `vendors` directory?

File: stylesheets/base/_typography.scss [hunk 2]
```diff
@@ -5,7 +5,7 @@
 /**
  * Import Google fonts
  */
-@import url('https://fonts.googleapis.com/css?family=Lato:400,700,800,900|Manuale:400,700');
+@import url('https://fonts.googleapis.com/css?family=Lato:400,700,800,900|Manuale:400,700|Material+Icons&display=swap');
```
Comment (helrond) [inline stylesheets/base/_typography.scss]: Do we need the `display=swap` thing? I'm reading it's the default now...

**Review by helrond — COMMENTED** [review]
Just a couple of questions. Overall looks good!

File: .stylelintrc.json (hunk 1 above)
Comment (HaSistrunk) [reply .stylelintrc.json]: Because that would be too easy...

File: stylesheets/base/_typography.scss (hunk 2 above)
Comment (HaSistrunk) [reply stylesheets/base/_typography.scss]: I'm seeing the same. Removing!

**Review by helrond — APPROVED** [review]

**Pull request merged**